waiter/integration/waiter/authentication_test.clj

```diff
@@ -311,7 +315,9 @@
     (if (jwt-auth-enabled? waiter-url)
       (let [waiter-host (-> waiter-url sanitize-waiter-url utils/authority->host)
             host (create-token-name waiter-url (rand-name))
-            service-parameters (assoc (kitchen-params) :name (rand-name))
+            service-parameters (assoc (kitchen-params)
+                                 :env {"USE_BEARER_AUTH" "true"}
```

Comment:
Can we use WAITER_BEARER_AUTH?

Reply:
`WAITER_*` environment variable names are reserved :(

https://github.com/twosigma/waiter/blob/0e6b8f406ae41076691e47fc2c06060d7d4383e0/waiter/src/waiter/service_description.clj#L41-L46